app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -152,11 +151,11 @@ Check with [assessment only training providers](/train-to-be-a-teacher/assessmen
 
 ### Other ways to get teaching work and experience
 
-[Supply teachers](https://www.tes.com/jobs/careers-advice/supply-teaching/how-become-supply-teacher) fill in when schools are temporarily short of staff. Register with a supply teacher agency for short-term roles in school.  
+Supply teachers provide teaching cover when schools are temporarily short of staff. Register with a supply teacher agency for short-term roles in school. [Find out more about being a supply teacher on the tes website](https://www.tes.com/jobs/careers-advice/supply-teaching/how-become-supply-teacher). 
 
-[Find your local authority ('council')](https://www.gov.uk/find-local-council) and then search their website for teaching roles.
+[Find your local authority (council)](https://www.gov.uk/find-local-council) and then search their website for teaching roles.
```

Comment:
i think we should just say 'council' - LA seems internal and GDS says to use council for user-facing services.